utils/utils_test.go

```diff
@@ -63,3 +63,84 @@ func assertSplitWithEscape(str string, expected []string, t *testing.T) {
 		t.Error("Unexpected string array built. Expected: `", expected, "` Got `", result, "`")
 	}
 }
+
+func TestCleanPath(t *testing.T) {
+	if IsWindows() {
+		parameter := "\\\\foo\\\\baz\\\\..\\\\bar\\\\*"
+		got := cleanPath(parameter)
+		want := "\\\\foo\\\\bar\\\\*"
+		if got != want {
+			t.Errorf("PrepareLocalPathForUpload(%s) == %s, want %s", parameter, got, want)
```

Comment:
Shouldn't *PrepareLocalPathForUpload* be replaced with *cleanPath*?
(In multiple occurrences in this function).